src/app/resources/resources.component.spec.ts

```diff
@@ -1,30 +1,124 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { FormsModule } from '@angular/forms';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { CouchService } from '../shared/couchdb.service';
 import { HttpModule } from '@angular/http';
 import { ResourcesComponent } from './resources.component';
-import { RouterModule } from '@angular/router';
-import { CouchService } from '../shared/couchdb.service';
+import { FormsModule }   from '@angular/forms'
 
 describe('ResourcesComponent', () => {
   let component: ResourcesComponent;
   let fixture: ComponentFixture<ResourcesComponent>;
+  let putSpy: any;
+  let getSpy: any;
+  let couchService;
+  let statusElement;
+  let de;
+  let testEvent;
+  let attachments;
+  let testdata;
+  let motoristshandbook;
+  let testresource1;
+  let testinput;
+  //let testArray;
+  //let blob;
 
-  beforeEach(async(() => {
+  beforeEach((() => {
     TestBed.configureTestingModule({
-      imports: [ FormsModule, RouterModule, HttpModule ],
+     
+      imports: [ FormsModule, RouterTestingModule, HttpModule ],
       declarations: [ ResourcesComponent ],
       providers: [ CouchService ]
     })
     .compileComponents();
```

Comment:
We actually don't need to use `.compileComponents` (I think it's created with the `ng generate` command) since we use Webpack.  That's why we don't need `async()`, either, which I see you've removed.